packages/libs/react-ui/src/components/Table/Table.stories.tsx

```diff
@@ -0,0 +1,107 @@
+import { ITable, Table } from '.';
+
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+
+const meta: Meta<
+  {
+    rowCount: number;
+    columnCount: number;
+  } & ITable
+> = {
+  title: 'Table',
+  argTypes: {
+    rowCount: {
+      control: { type: 'range', min: 1, max: 6, step: 1 },
+    },
+    columnCount: {
+      control: { type: 'range', min: 1, max: 6, step: 1 },
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<
+  {
+    rowCount: number;
+    columnCount: number;
+  } & ITable
+>;
+
+/*
+ *👇 Render functions are a framework specific feature to allow you control on how the component renders.
+ * See https://storybook.js.org/docs/7.0/react/api/csf
+ * to learn how to use render functions.
+ */
+
+export const Primary: Story = {
+  name: 'Table',
+  args: {
+    rowCount: 3,
+    columnCount: 5,
+  },
+  render: ({ rowCount, columnCount }) => {
+    return (
+      <Table>
+        <Table.Head>
+          <Table.Tr>
+            {Array.from(Array(columnCount)).map((id, tdIdx) => {
+              return <Table.Tr.Th key={`td${tdIdx}`}>test {tdIdx}</Table.Tr.Th>;
```

Comment:
I think keeping subcomponents at one level is best. So `Table.Th` should be sufficient